<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramDlgClass</class>
 <widget class="QDialog" name="ProgramDlgClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>245</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>245</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Program</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>USB Device Located</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lightUSBDeviceLocated">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="generichid.qrc">:/GenericHID/redlight</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>HID Device Mode</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lightHIDDeviceMode">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="generichid.qrc">:/GenericHID/redlight</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Bootloader Mode</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lightBootloaderMode">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="generichid.qrc">:/GenericHID/redlight</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnStartBootloader">
         <property name="text">
          <string>Start Bootloader</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnProgram">
         <property name="text">
          <string>Program</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRestartDevice">
         <property name="text">
          <string>Restart Device</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="generichid.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDlgClass</receiver>
   <slot>onClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRestartDevice</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDlgClass</receiver>
   <slot>onRestartDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProgram</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDlgClass</receiver>
   <slot>onProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartBootloader</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDlgClass</receiver>
   <slot>onStartBootloader()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRefresh</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDlgClass</receiver>
   <slot>onRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClose()</slot>
  <slot>onRestartDevice()</slot>
  <slot>onProgram()</slot>
  <slot>onStartBootloader()</slot>
  <slot>onRefresh()</slot>
 </slots>
</ui>
